File: AoVivoAprendaJavaZero/app/src/main/java/com/app/aovivoaprendajavazero/MainActivity.java
package com.app.aovivoaprendajavazero;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.app.aovivoaprendajavazero.classes.Cliente;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //Pego o que o usuário digitou
        String nome = "danillo";//Peguei o que o usuário digitou na tela

        Cliente cliente = new Cliente();
        double saldo = cliente.recuperarSaldo();

    }
}

File: AulaModulos/lojas/src/main/java/com/danilloteles/lojas/MainActivity.kt
package com.danilloteles.lojas

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.danilloteles.validacoes.Valida

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Valida.validarEmail()

    }
}